### ISSUE TITLE
Cant't run the application after installing the gem

### ISSUE THREAD

**Issue by K-Sato1995** [bitjourney/graphql-autotest#5]
Hi.
I got the following error when I tried to run the application after adding the gem to the project.
Tried on several projects and got the same error.

Is there anything else I'm supposed to do besides adding the gem to Gemfile and running `bundle install` to use the gem??
Am I missing something obvious??

```
/Rails/graphql_api/vendor/bundle/ruby/2.4.0/gems/graphql-autotest-0.1.1/lib/graphql/autotest/field.rb:3:in `new': {:keyword_init=>true} is not a symbol (TypeError)
```

**Comment by pocke** [comment]
Thanks for reporting!
`keyword_init` option of `Struct#new` is available only with Ruby 2.5+, so this gem actually needs Ruby 2.5+.
https://bugs.ruby-lang.org/issues/11925


So I'll update the required_ruby_version to require Ruby 2.5+. If you'd like to use this gem, please update Ruby version :pray:

**Comment by K-Sato1995** [comment]
@pocke 
Thanks for the quick response! 
I installed the gem on a different project that uses `ruby2.5+` and could successfully run the project.
